src/cargo/ops/mod.rs

```diff
@@ -44,7 +44,6 @@ pub(crate) mod cargo_compile;
 pub mod cargo_config;
 mod cargo_doc;
 mod cargo_fetch;
-mod cargo_generate_lockfile;
```

Comment:
Thank you for splitting this out as it makes things much clearer.  I think there was something I had overlooked.  I thought you split `cargo_generate_lockfile` rather than straight rename.  It feels odd to favor one name over the other for the command.

What is your motivation for renaming?  `cargo_generate_lockfile` is so small I wonder if its a separate file.  I had considered suggesting it `lockfile.rs` but we appear to have that.  Maybe its worth being in there still?

Thoughts?

Reply:
Thinking more on this, the primary thing people look for is `update` and not `generate_lockfile`.

Also, adding `Cargo.toml` editing makes this so `resolver` and `lockfile` names are less applicable.

Seems like this rename is fine for now.  We can evolve it as the programs evolve.